banksystem/accounts/views.py

```diff
@@ -1,22 +1,23 @@
-from django.shortcuts import render
-from django.views.generic import TemplateView
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
 from .utils import get_user_bank_accounts
+from .serializer import AccountSerializer
```

Comment:
create an APView which returns the detail of the account being requested.

"/accounts/1/"

when a user try to access an account which doesnt belong to them, return an error. otherwise return user's accoun

Reply:
do this with a permission class